parts/k8s/kubernetesinstalls.sh

```diff
@@ -129,6 +129,27 @@ installDockerEngine() {
     fi
 }
 
+configureDockerEngine() {
+    echo "stopping docker engine"
+    sudo systemctl stop docker
+    cat << EOF > /tmp/daemon.json 
+{
+    "storage-driver": "overlay2",
+    "live-restore": true,
+    "log-driver": "json-file",
+    "log-opts":  {
+        "max-size": "50m",
+        "max-file": "5"
+    }
+}
+EOF
+    sudo mv /tmp/daemon.json /etc/docker/daemon.json
+    echo "cleaning up aufs storage"
+    sudo rm -rf /var/lib/docker/aufs
```

Comment:
I'm curious: in the "docker-engine" VHD CI build scenario, moby (or no other docker) is installed *except* docker-engine. How does `/var/lib/docker/aufs` get there? Is it installed by default by Ubuntu 16.04-LTS (that would seem unlikely).

Reply:
I think the docker is installed as part of this step installDockerEngine (in packer/install-dependencies.sh), which is used for pulling down the images.

Reply:
We use the same function to install docker-engine in non-VHD scenarios (CSE calls installContainerRuntime which calls installDockerEngine) so how is it different for VHDs specifically?